xclim/sdba/adjustment.py

```diff
@@ -1209,19 +1378,24 @@ def _adjust(
         return out
 
 
-class OTC:
+class OTC(MultivariateAdjust):
     """OTC"""
 
     @classmethod
     def _adjust(
         cls,
-        ref,
-        hist,
-        bin_width=None,
-        bin_origin=None,
-        numItermax=100_000_000,
+        ref: xr.DataSet,
+        hist: xr.DataSet,
+        sim: None = None,  # TODO : ignored
```

Comment:
With the `MultivariateAdjust` class, I still need to list `sim` as input, otherwise `bin_width` gets set to `sim=hist`.
I'll see if I can think of something to make it look better.

Reply:
If I use this signature: 

```python
class OTC(MultivariateAdjust):
    """OTC"""

    @classmethod
    def _adjust(
        cls,
        ref: xr.DataSet,
        hist: xr.DataSet,
        sim: xr.DataSet | None = None,
        *,
        sim_is_hist: bool = True, 
        bin_width: list | None = None,
        bin_origin: list | None = None,
        numItermax: int = 100_000_000,
        group: str | Grouper = "time",
    ):
        if sim_is_hist == False:
            raise ValueError("OTC does not take a `sim` argument")
...
```

it seems to work. But I think we can simply use an attrs in `sim` instead, let me rewrite this